apps/explorer/lib/explorer/chain/transaction.ex

```diff
@@ -1363,8 +1363,8 @@ defmodule Explorer.Chain.Transaction do
   @doc """
   Builds an `Ecto.Query` to fetch transactions by hashes
   """
-  @spec transactions_by_hashes([Hash.t()]) :: Ecto.Query.t()
-  def transactions_by_hashes(hashes) do
+  @spec transactions_by_hashes_query([Hash.t()]) :: Ecto.Query.t()
```

Comment:
very much imho nitpick: if we follow a shorter naming in other modules, can we rename this one as well (for consistency)?

```suggestion
  @spec by_hashes_query([Hash.t()]) :: Ecto.Query.t()
```